packages/qgrid-ngx/src/lib/resize/resize.directive.ts

```diff
@@ -106,7 +106,7 @@ export class ResizeDirective implements OnInit, OnDestroy {
     const { context, path, key } = this;
     const { layout } = this.model;
 
-    const state = clone(layout()[path]);
+    const state = clone(layout()[path as keyof typeof layout]);
```

Comment:
you can use `path as keyof LayoutState`, just a bit shorter

Reply:
Doesn't work.